README.md

```diff
@@ -10,13 +10,13 @@ The goal of [this project](https://github.com/orgs/project-origin/projects/1) is
 The registry will enable the following:
 
 - [ ] Data in the registry is publicly verifiable.
-- [ ] Enable the issuance of Granular Certificates.
-- [ ] Privacy of the individual quantities is hidden behind a Pedersen-commitments.
-- [ ] Enable the claim of Granular Certificates to some consumption.
+- [x] Enable the issuance of Granular Certificates.
+- [x] Privacy of the individual quantities is hidden behind a Pedersen-commitments.
+- [x] Enable the claim of Granular Certificates to some consumption.
```

Comment:
An open question - should we add that the user get a receipt with the subproof and address of the roothash for offline verification of the transaction ?

Reply:
I think it is in the following goal:

> Users can verify that they have a provable quantity of a Granular Certificate, that mathematically proves it is only used once.

We need to create the necessary endpoints that enable a user to get merkleproofs for events, and the series of events for a FederationCertificateId.